modules/govuk_jenkins/templates/jobs/run_rake_task.yaml.erb

```diff
@@ -37,8 +37,8 @@
             - draft_frontend
             - email_alert_api
             - frontend
-            - router_backend
```

Comment:
`locations_api` should be added here

Reply:
🤦  sorry. Thanks. Clearly need another coffee!